pulsar-metadata/src/main/java/org/apache/pulsar/metadata/api/MetadataCache.java

```diff
@@ -155,4 +155,12 @@
      * @param path the path of the object in the metadata store
      */
     void refresh(String path);
+
+
+    /**
+     * Check if the value of this path is already in cache.
+     * @param path the path of the object in the metadata store
+     * @return true if the path exists.
+     */
+    boolean existsInCache(String path);
```

Comment:
I wouldn't add a new method in the API that we only are going to be using internally.

Two solutions: 
 1. There is already `MetadataCache.getIfCached()`
 2. `MetadaCache.refresh()` implementation should take care of it internally.

Reply:
The only reason I am not using `MetadataCache.getIfCached()` is it would block and wait for current load future is completed. 

I am taking solution 2.  It would be "refresh" if we have not already loaded the path object. right?